nginx1.12/healthcheck.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
```

Comment:
Added nginx config check

```
# Check nginx config is valid
nginx -t || exit 1
```

Reply:
if there are errors in the configuration file, the service simply does not start. Respectively the pid file will not be found. Is this check really necessary?

Reply:
There may be a case when nginx was already running and it's configuration is being reloaded with a broken config. It will continue to run using the old config. Also, this check does not add any overhead, so definitely won't hurt.